### ISSUE TITLE
About the Repo

### ISSUE THREAD

**Issue by ZeroDayOwl** [phoebeireland/fencingtracker#1]
Hey, the name of the repo sounds interesting. Is it about keeping track of Fencing points in bouts?

**Comment by phoebeireland** [comment]
Yep! And be a notebook of sorts for fencers to keep notes on matches and a place to store and compare competition results too. If I can eventually get the functionality in too, it'll maybe track some other fencing things as well or open it to have a sort of blog functionality too.

**Comment by phoebeireland** [comment]
But, as you can see there's currently nothing in this repo 😂 will start working on it soon.

**Comment by ZeroDayOwl** [comment]
>Yep! And be a notebook of sorts for fencers to keep notes on matches and a place to store and compare competition results too. If I can eventually get the functionality in too, it'll maybe track some other fencing things as well or open it to have a sort of blog functionality too.

Sounds interesting!

> But, as you can see there's currently nothing in this repo joy will start working on it soon.

Haha it's alright! I had thought of starting Fencing last year, but couldn't do much because of this COVID-19, but hoping to continue from the next year. Guess I might be able to contribute to it too :crossed_swords: !

**Comment by phoebeireland** [comment]
Sure thing! I'll be starting it soon, so feel free to point out all the issues you find!

**Comment by ZeroDayOwl** [comment]
Sure!
